recipes/fastjet/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "fastjet" %}
+{% set version = "3.4.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastjet-{{ version }}.tar.gz
+  sha256: 66ad0bf9c9db157264733553bc133aba3c7a5f1bb51939ee653695ada0bcdd94
+  patches:
+    - 0001-Don-t-overwrite-external-CXXFLAGS-and-LDFLAGS.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  missing_dso_whitelist:
+    - $RPATH/libfastjet.so.0
+    - $RPATH/libfastjettools.so.0
```

Comment:
You probably shouldn't be using this, if the CI fails because of it there is probably a bug.

Reply:
@chrisburr can you give some references on how to debug overlinking errors that are releated to things similar to this? I'm not experienced here, but that seems to be the only thing that is fialing in my local tests.